anomalib/models/components/filters/blur.py

```diff
@@ -19,32 +31,37 @@ class GaussianBlur2d(nn.Module):
 
     def __init__(
         self,
-        kernel_size: Union[Tuple[int, int], int],
         sigma: Union[Tuple[float, float], float],
-        channels: int,
+        channels: int = 1,
+        kernel_size: Optional[Union[Tuple[int, int], int]] = None,
         normalize: bool = True,
         border_type: str = "reflect",
         padding: str = "same",
     ) -> None:
         """Initialize model, setup kernel etc..
 
         Args:
-            kernel_size (Union[Tuple[int, int], int]): size of the Gaussian kernel to use.
             sigma (Union[Tuple[float, float], float]): standard deviation to use for constructing the Gaussian kernel.
```

Comment:
Not part of this PR but should we move these init_args above to make it consistent with other classes?